src/boot/bootctl.c

```diff
@@ -491,6 +520,33 @@ static int parse_argv(int argc, char *argv[]) {
                         arg_dry_run = true;
                         break;
 
+                case ARG_SECURE_BOOT_AUTO_ENROLL:
+                        arg_secure_boot_auto_enroll = true;
```

Comment:
hmm, can we change this to take an argument please? i.e. parse_boolean_argument().

I sense sooner or later we want to define a standard path for these SB keys, and if we have that we might want to enable this by default if we find keys but disable if we don't. hence we might want to have an explicit way to disable this, hence `--secure-boot-auto-enroll=no` might be our way out.